source/tutorials/php.rst

```diff
@@ -0,0 +1,128 @@
+.. _php:
+
+Installing and Configuring PHP
+******************************
+
+This tutorial describes how to configure and use PHP and PHP extensions on |CL-ATTR|. Although we specifically address PHP this tutorial can serve as a general guide for working with applications in the |CL| :ref:`stateless` environment.
+
+We will focus on enabling PHP and PHP extensions in |CL|.  For a complete :abbr:`LAMP (Linux, Apache, MySQL, PHP)` setup in |CL| refer to the :ref:`web-server-install` section of the :ref:`wordpress` tutorial. For this tutorial you do not need to install MariaDB or WordPress.
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Prerequisites
+*************
+
+* :ref:`Install <bare-metal-install-desktop>` |CL| on your host system
+* Follow :ref:`web-server-install` to install Apache\* and PHP
```

Comment:
A webserver isn't required to use PHP. The prereq here is have the `php-basic` bundle installed.

Reply:
I want to leave this in -- people who don't use httpd or other webserver will disregard.  But using httpd gives the opportunity to use a browser to look at phpinfo(), which is useful for people who are following external instructions for modifying php.ini  I added a note in the latest commit indicating httpd is not a requirement.

Reply:
I thought it over during lunch.  You're right. I've removed the httpd references/requirement and added a note with a pointer to the LAMP install tutorial if that's what people are looking for. This is just PHP & PHP-FPM now.